connector/countconnector/generated_package_test.go

```diff
@@ -1,5 +1,4 @@
-// Copyright The OpenTelemetry Authors
-// SPDX-License-Identifier: Apache-2.0
+// Code generated by mdatagen. DO NOT EDIT.
```

Comment:
Don't we need the copyright at the top of this file?

Reply:
None of the code generated by mdatagen has the copyright, not sure what the right thing to do is here